index.ts

```diff
@@ -5,6 +5,10 @@ declare module 'vue' {
     interface ComponentCustomProperties {
         $t: TFunction;
         $i18next: i18n;
+        // Optional custom pattern for matching slot start of the `TranslationComponent. Internal.
```

Comment:
To me it seems better to not have these internal properties be part of the public `ComponentCustomProperties` type, as I can't think of a use-case for them in user components, We can just use the internal propertie(s) without declaring them.